src/Illuminate/Routing/UrlGenerator.php

```diff
@@ -128,6 +128,17 @@ public function current()
     }
 
     /**
+     * Check if the current url have a specific keyword.
+     *
+     * @param  string $keyword
+     * @return true|false
+     */
+    public function has($keyword)
+    {
+        return strpos($this->current(), $keyword);
```

Comment:
Quoting the php doc about strpos:
> This function may return Boolean FALSE, but may also return a non-Boolean value which evaluates to FALSE. Please read the section on Booleans for more information. Use the === operator for testing the return value of this function.

So, if you want to return a boolean for your `has` method, you have to do something like:

```
 return strpos($this->current(), $keyword) !== false;
```